src/main/java/com/compilercharisma/chameleonbusinessstudio/config/SecurityConfiguration.java

```diff
@@ -20,6 +20,7 @@ public SecurityWebFilterChain securityWebFilterChain(
         return security
                 .oauth2Login(Customizer.withDefaults())
                 .authorizeExchange()
+                .pathMatchers("/api/v1/users/**").permitAll()
```

Comment:
Is this intentional?

Reply:
Ooop! Not it isn't that was for manual testing purposes, gonna take it off